book/src/user/simple-example.md

```diff
@@ -0,0 +1,19 @@
+# A simple example
+
+Let's start with a simple circuit, to introduce you to the common APIs and how they are
+used. The circuit will take a public input $c$, and will prove knowledge of two private
+inputs $a$ and $b$ such that
+
+$$a * b = c.$$
+
+```rust
+# extern crate halo2;
+use halo2::arithmetic::FieldExt;
+
+struct MyCircuit<F: FieldExt> {
```

Comment:
```suggestion
struct MyInstance<F: FieldExt> {
```

Reply:
In the current API (which the book will reflect as we develop it), this struct is what will implement the `Circuit` trait, so it is accurately named.